chombot-common/src/tournaments_watcher/ema.rs

```diff
@@ -213,7 +214,12 @@ pub async fn get_rcr_tournaments() -> Result<Tournaments, TournamentsFetchError>
 }
 
 pub async fn get_tournaments() -> Result<Tournaments, TournamentsFetchError> {
-    let body = reqwest::get(CALENDAR_URL)
+    let body = reqwest::Client::builder()
+        .user_agent(USER_AGENT)
+        .build()
```

Comment:
Can we move this and the `USER_AGENT` constant to a common file?

Reply:
sure thing